dbt/models/marts/schools/dim_school_status.sql

```diff
@@ -105,6 +109,7 @@ full_status as (
             , 0
         ) as ever_active_before,
         (is_active || prev_year_active || ever_active_before) status_code,
```

Comment:
let's avoid concat strings for labeling going forward-- for your final status sorting, you can use case statements as such (see line 124). also, when building flags, the convention is to use `is_` to signify to other developers it is a binary flag. so you could have the following new fields: `is_active_last_year`. for `ever_active_before` you can just say `max(is_active) = 1`, the rows bounded stuff is just make more data for the query to read when it has all it needs.

```suggestion
    case 
        when coalesce(is_active,prev_year_active,ever_active_before) = 1 then 'active - retained' ... 
```